tests/test_helpers.py

```diff
@@ -76,6 +76,14 @@ def test_escape_markdown_v2_text_link(self):
             test_str, version=2, entity_type=MessageEntity.TEXT_LINK
         )
 
+    def test_escape_markdown_v2_custom_emoji(self):
+        test_str = "tg://emoji?id=536832))\\4170671202286"
```

Comment:
does this work?

Reply:
it does, but it is not in the update, so I will remove it